workspaces/github-actions/plugins/github-actions/src/alpha/entityContent.tsx

```diff
@@ -13,22 +13,35 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { convertLegacyRouteRef } from '@backstage/core-compat-api';
+import {
+  compatWrapper,
+  convertLegacyRouteRef,
+} from '@backstage/core-compat-api';
 import { EntityContentBlueprint } from '@backstage/plugin-catalog-react/alpha';
 import { isGithubActionsAvailable } from '../components/Router';
 import { rootRouteRef } from '../routes';
 
 /**
  * @alpha
  */
-export const entityGithubActionsContent = EntityContentBlueprint.make({
-  name: 'entity',
-  params: {
-    path: 'github-actions',
-    title: 'GitHub Actions',
-    filter: isGithubActionsAvailable,
-    routeRef: convertLegacyRouteRef(rootRouteRef),
-    loader: () =>
-      import('../components/Router').then(m => <m.Router view="table" />),
-  },
-});
+export const entityGithubActionsContent =
+  EntityContentBlueprint.makeWithOverrides({
+    config: {
+      schema: {
+        layout: z => z.enum(['table', 'cards']).default('table'),
+      },
+    },
+    name: 'github-actions-entity-content',
```

Comment:
Oh, I realized this name should go away. The previous `'entity'` doesn't really make sense either tbh and I think it's worth shipping a breaking change, especially since we're adding config with this.

The default full ID for this extension without a name is `entity-content:github-actions`. If we set the name to this the full name is gonna be `entity-content:github-actions/github-actions-entity-content` 😅